src/gwt/src/org/rstudio/studio/client/application/ApplicationQuit.java

```diff
@@ -146,7 +146,7 @@ else if (workbenchContext_.isServerBusy() && terminalHelper_.warnBeforeClosing(b
             globalDisplay_.showYesNoMessage(
                   MessageDialog.QUESTION,
                   caption, 
-                  msg + " Are you sure you want to quit?",
+                  msg + " " + constants_.applicationQuitMessage(),
```

Comment:
For flexibility in localization, would run this through a placeholder string, e.g.:

{0} Are you sure you want to quit?